Exercises/ex-03/base/remain.py

```diff
@@ -0,0 +1,114 @@
+# Prompts the user to enter the following information for a customer contact list:
+    # Last name
+    # First name
+    # Phone number
+    # Email address
+# Stores the supplied information in a dictionary with appropriately named keys
+# Once stored, prints out the entry in a format that looks good to you
+
+
+# Each Dict is one record. Lname, Fname, Phone, Email, and recordnum are keys.
+# Each Dict identifier is in a master list.
+
+import sys
+
+def search():
+    print("SEARCH is not available at this time.\n\n")
+    main()
+
+def verifyphone(p):
+    #initialize starting input:  p
+    startnum = p
+    #initialize an empty string to catch the number we need.
+    outnum = ""
+    #goodnums is a list of just the digits we want to accept.
+    goodnums = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"]
+
+    #for loop goes through their string and adds only the digits of goodnums to the output variable
+    for num in startnum:
+        if num in goodnums:
+            outnum += num
+        else:
+            continue
+
+    if len(outnum) > 10:
+        outnum = "Error:  Too many numbers."
+    else:
+        outnum = "({}) {}-{}".format(outnum[:3],outnum[3:6],outnum[6:])
+
+    return outnum
+
+def verifyemail(e):
+    #slice off the last four letters
+    dotcom = e[-4::1]
+    
+    #list of Top Level Domains that don't suck
+    tld_list = (".com",".org",".net",".int",".edu",".gov",".mil")
+
+    #check for @ and .something
+    if "@" not in e:
```

Comment:
It would be just as easy to count the number of @s and check for 1.

Reply:
So i did that. get off my back.